test/cli/autocorrect_and_and/autocorrect_and_and.rb

```diff
@@ -0,0 +1,26 @@
+# typed: true
+
+class A
+  extend T::Sig
+
+  sig {returns(T.nilable(Integer))}
+  def foo; nil; end
+
+  def bar
+    a = A.new
+
+    if a.foo && a.foo.even?
```

Comment:
Mildly interesting corner case you might test (maybe this isn't the right spot for it): if there's whitespace before `even?`, e.g. `a.foo.<NL>even?` (I think the autocorrect should not fire since you're looking for a period immediately preceding.)

Reply:
great test!